src/hypertable_cache.c

```diff
@@ -55,7 +55,7 @@ hypertable_cache_create()
 			.hcxt = ctx,
 		},
 		.name = "hypertable_cache",
-		.numelements = 16,
+		.numelements = _MAX_CATALOG_TABLES,
```

Comment:
cc @cevian

Reply:
ping @cevian (just keeping this fresh)

Reply:
also @gayyappan what's the reason for this change

Reply:
hard coded. Need to change whenever we add new catalog tables?

Reply:
This has no relation to the number of catalog tables I don't think.

Reply:
This change should be removed

Reply:
removed.